tests/integration/bootstrap.sh

```diff
@@ -96,7 +105,7 @@ main() {
     set -e
     mkdir -p ${RUNTIME_DIR}
 
-    if [ ! -d ${OTBR} ]; then
+    if [[ $SKIP_BUILDING_OTBR == 0 ]] && [ ! -d ${OTBR} ]; then
```

Comment:
Is mixing `[[ ]]` and `[]` intended?

Reply:
I actually want arithmetic comparison here, changed to `(())`.